### DISCUSSION TITLE
Will Aldente work if I unplug My M2 MacBook Air at night?

### DISCUSSION THREAD

**Discussion by lylydang** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#838] [Q&A] [answered]
I've been trying to do more researches for my question but I still cannot find the answer, hence posting it here.
I'm a newbie with Aldente first of all. I installed Aldente on my M2 MacBook Air since I got it, but still keeping the habit of unplugging it at night (I plug it in all the time during the daytime as I work though). Will this behavior interfere with Aldente's features? Should I just keep my MB plugged in all day and overnight? Thanks.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @lylydang,

Unplugging your MacBook will not interfere with AlDente in general. So you can definitely unplug it.

Cheers,
Matthias
